internal/uvm/create.go

```diff
@@ -315,3 +314,16 @@ func (uvm *UtilityVM) normalizeMemorySize(ctx context.Context, requested int32)
 func (uvm *UtilityVM) DevicesPhysicallyBacked() bool {
 	return uvm.devicesPhysicallyBacked
 }
+
+// Closes the external GCS connection if it is being used and also closes the
+// listener for GCS connection.
+func (uvm *UtilityVM) CloseGCSConnection() error {
+	var err error = nil
```

Comment:
Default value of an error is already nil so can be removed, or just change the return in the declaration to (err error) and remove the 'var err error = nil' altogether.